engine/src/main/java/org/terasology/engine/logic/console/commands/CoreCommands.java

```diff
@@ -479,7 +479,7 @@ public String dumpEntities(@CommandParam(value = "componentNames", required = fa
         PrefabSerializer prefabSerializer =
                 new PrefabSerializer(engineEntityManager.getComponentLibrary(), engineEntityManager.getTypeSerializerLibrary());
         WorldDumper worldDumper = new WorldDumper(engineEntityManager, prefabSerializer);
-        Path outFile = PathManager.getInstance().getHomePath().resolve(Instant.now() + "-entityDump.json");
+        Path outFile = PathManager.getInstance().getHomePath().resolve(Instant.now().toString().replaceAll(":", "-") + "-entityDump.json");
```

Comment:
would it not be cleaner to use the iso-8601 basic format in general, and not the extended format? see:
https://stackoverflow.com/questions/27725408/alternative-to-colon-in-a-time-format